pkg/backend/net/net.go

```diff
@@ -41,8 +41,8 @@ func (e ErrHTTPCode) Error() string {
 	}
 }
 
-// Backend represents a host that accepts requests for metrics over HTTP.
-type Backend struct {
+// NetBackend represents a host that accepts requests for metrics over HTTP.
+type NetBackend struct {
```

Comment:
I suppose It makes sense to change this to `HTTPBackend`.

Reply:
I've considered that name. I discarded it, because it does not differentiate from the gRPC implementation which is also HTTP2.